workflow-to-js-transformer/src/main/java/com/github/cafdataprocessing/workflow/transform/WorkflowTransformer.java

```diff
@@ -66,7 +66,7 @@ private WorkflowTransformer(){}
     public static String retrieveAndTransformWorkflowToJavaScript(long workflowId, String projectId, String processingApiUrl)
             throws ApiException, WorkflowTransformerException, WorkflowRetrievalException {
         final String workflowAsXML = retrieveAndTransformWorkflowToXml(workflowId, projectId, processingApiUrl);
-        return transformXmlWorkflowToJavaScript(workflowAsXML);
+        return transformXmlWorkflowToJavaScript(workflowAsXML, projectId);
```

Comment:
Knowing that we're going to also be passing tenantId here in the future I think this should be a map of parameter name to value. I'm not too fussed on it being part of this changeset but I think that is how this should end up by the time tenantId is added.